src/Arcus.Messaging.Pumps.Abstractions/Resiliency/DefaultMessagePumpCircuitBreaker.cs

```diff
@@ -44,25 +43,62 @@ public virtual async Task PauseMessageProcessingAsync(string jobId, Action<Messa
         {
```

Comment:
I don't understand - the code that we have here is completely different as opposed to what we had here:
https://github.com/arcus-azure/arcus.messaging/pull/453/files#diff-397dfb045522acf325b8c768be9bf7b7b69846d6118ce9bba9f57c96879b3de0

Reply:
When I pull the repo and look at it locally, the file looks as expected though.

Reply:
It's the same now, no? Think it's correct?

Reply:
Yes it's OK.